python/felis/tap.py

```diff
@@ -60,7 +42,30 @@ def init_tables(
     tap_keys_table: str | None = None,
     tap_key_columns_table: str | None = None,
 ) -> MutableMapping[str, Any]:
-    """Generate definitions for TAP tables."""
+    """Generate definitions for TAP tables.
+
+    Parameters
+    ----------
+    tap_schema_name
+        Name of the TAP schema.
+    tap_tables_postfix
+        Postfix for table names.
+    tap_schemas_table
+        Name of the schemas table.
+    tap_tables_table
+        Name of the tables table.
+    tap_columns_table
+        Name of the columns table.
+    tap_keys_table
+        Name of the keys table.
+    tap_key_columns_table
+        Name of the key columns table.
+
+    Returns
+    -------
+    tables : `dict`
```

Comment:
Sphinx doesn't work out the return type?

Reply:
From what I understand, the version of numpydoc which I am using does not correctly infer return types and they have to be manually included.